dapr-spring/dapr-spring-cloudconfig/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.dapr.spring</groupId>
+        <artifactId>dapr-spring-parent</artifactId>
+        <version>0.15.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>dapr-spring-cloudconfig</artifactId>
+    <name>dapr-spring-cloudconfig</name>
+    <description>Dapr Spring Cloud Config</description>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.dapr.spring</groupId>
+            <artifactId>dapr-spring-boot-autoconfigure</artifactId>
+            <version>${project.parent.version}</version>
+            <scope>compile</scope>
+        </dependency>
+
+<!--        <dependency>-->
```

Comment:
@lony2003 we can remove the comments.. there is no need to keep them

Reply:
removed.